### PR TITLE
Add a menubar application to toggle AutoRaise and adjust preferences

### PR DESCRIPTION
**Pull request by lhaeger**

### CODE REVIEW

File: AutoRaise.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (sbmpost) [inline AutoRaise.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:5]: I assume this is not enforcing 32 bit or something? Just a warning?

File: AutoRaise.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+/* Begin PBXBuildFile section */
+		C501B69525875F32005B3964 /* LICENSE.md in Resources */ = {isa = PBXBuildFile; fileRef = C501B69425875F32005B3964 /* LICENSE.md */; };
+		C557DD431B988ED300E59BC9 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = C557DD421B988ED300E59BC9 /* AppDelegate.swift */; };
+		C557DD451B988ED300E59BC9 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = C557DD441B988ED300E59BC9 /* Images.xcassets */; };
+		C557DD481B988ED300E59BC9 /* MainMenu.xib in Resources */ = {isa = PBXBuildFile; fileRef = C557DD461B988ED300E59BC9 /* MainMenu.xib */; };
+		C59A84F02586387800AD9D56 /* AutoRaise in Resources */ = {isa = PBXBuildFile; fileRef = C59A84EF2586387800AD9D56 /* AutoRaise */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXFileReference section */
```
Comment (sbmpost) [inline AutoRaise.xcodeproj/project.pbxproj:17]: no need to add the README.md? I suggest you add a special section to the README on how to start AutoRaise with your launcher and how to compile it from the command line.

File: AutoRaise.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+				DEVELOPMENT_TEAM = 8B7L8R4FBJ;
+				INFOPLIST_FILE = Launcher/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks";
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
+				MARKETING_VERSION = 0.0.1;
+				ONLY_ACTIVE_ARCH = YES;
+				PRODUCT_BUNDLE_IDENTIFIER = com.brummelhook.AutoRaise;
+				PRODUCT_NAME = AutoRaise;
+				PROVISIONING_PROFILE = "";
+				SWIFT_OBJC_BRIDGING_HEADER = "AutoRaise-Bridging-Header.h";
```
Comment (sbmpost) [inline AutoRaise.xcodeproj/project.pbxproj:296]: I wonder what this brings as the bridging file is basically empty.

File: Launcher/AppDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    @IBOutlet weak var delaySlider: NSSlider!
+    @IBOutlet weak var enableWarpButton: NSButton!
+    // About
+    @IBOutlet weak var aboutText: NSTextField!
+    @IBOutlet weak var homePage: NSButton!
+    
+
+    let appAbout =  "AutoRaise\n" +
+        "Version 1.0, 2020-12-13\n\n" +
+        "©2020 Stefan Post, Lothar Haeger\n" +
```
Comment (sbmpost) [inline Launcher/AppDelegate.swift:41]: haha, cool :-)

**Review by sbmpost — APPROVED** [review]
Looks good 👍, some small comments below.

File: AutoRaise.xcodeproj/project.pbxproj (hunk 3 above)
Comment (lhaeger) [reply AutoRaise.xcodeproj/project.pbxproj:296]: That's left from when I once pulled in your code into the xcode project, which auto-generated the bridging header as well. Needs to be cleaned up.

File: AutoRaise.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (lhaeger) [reply AutoRaise.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:5]: I compiled both binaries on Catalina to 64 bit, not enforcing anything, just a log of processed checks according to https://stackoverflow.com/questions/50367896/ideworkspacechecks-plist-file-suddenly-appear-after-updated-xcode?noredirect=1&lq=1

**Pull request closed without merging**